### ISSUE TITLE
Findable Help SC3.2.6 - same relative order

### ISSUE THREAD

**Issue by guyhickling** [w3c/wcag#1230]
The SC says:

> .....at least one option is included in the same relative order on each page

This phrase does not make logical or literary or meaningful sense because one option, by definition, cannot on its own be in the same relative order:
1) You need two or more of something for them to have an "order". 
2) And they can only be in the "same relative" order when compared with something else, but nothing else is mentioned here so the reader is left confused.

However if we simply replace "same relative order" by its glossary definition (instead of linking to it), then the words make much more sense. What has happened here is that, to reduce the SC by just 3 words, the meaning has been obscured. A case of sacrificing meaningful comprehension for mere brevity. 

So could the above phrase be changed to:

> .....one option is included, in the same position relative to other items, on each page:

(that uses the words from the glossary)? Alternatively just replace "order" by "position". That also makes sense:

> .....one option is included in the same relative position on each page:

**Comment by scha14** [comment]
https://www.w3.org/WAI/WCAG22/Understanding/findable-help.html#dfn-same-relative-order defines relative order as 

> same position relative to other items

and is linked to _relative order_ in the SC. Is this okay to close?

**Comment by guyhickling** [comment]
I know it links to the glossary, but the only other SC to use that glossary definition, 3.2.3, uses it in a way that is grammatical and therefore makes more sense. But the same phrase is ungrammatical in this SC, making the SC that much harder to understand. I don't see the need to use a glossary definition just because it happens to exist for something else. That just forces people to look it up to find out what it means.

In this case, I think simply replacing the word "order" by the word "position" makes it more meaningful, and grammatical, at the same time.

**Comment by scha14** [comment]
Draft response : 

> That applies to all definitions. It is always going to be a tradeoff between keeping the SC concise with links vs having everything in one place. The link in this case is easily accessible and having fewer words seems like an okay tradeoff.

**Comment by alastc** [comment]
Suggesting update to the draft response:
> The current language is "one option is included in the same relative order on each page". So it is clear that the "option" is relative to other things on the page. In the same way that 3.2.3 Consistent Navigation puts the nagivation in the same relative order on each page.
> "Position" as a term was considered but not thought suitable because it implies a 2 dimensional aspect that may not always apply. For example, at different viewport sizes the position/location would not be consistent. The only fair and testable word that seemed to fit was "relative order".

NB: When this goes to the group, the other option is to change the text.

**Comment by guyhickling** [comment]
> So it is clear that the "option" is.......

It is clear to you, because you and others in the Group have been working on this for a while so are familar with it. Complex things always look much easier when we have been involved with them for some time.

Conversely, however, other people are now beginning to comment on how confusing this SC is in its wording. Just take a look at #1287 and #1314. In those two issues three more people have all commented on its lack of clarity, and another gave one of those comments the thumbs up. (And there are probably other # issues that I've missed.) 

And these are all people well familiar with the WCAG. It seems obvious this SC isn't as clear to other people as you think it is. Perhaps it is time to take a serious assessment of it and reword the thing before it causes even more confusion out in the web dev world after release?

**Comment by alastc** [comment]
The group discussed this today, and the same discussion and decision followed: position implies too much given the various scenarios.
The response above was accepted.
